newsfragments/1044.removal.rst

```diff
@@ -0,0 +1,17 @@
+Most of the public classes that Trio exports – like `trio.Lock`,
+`trio.SocketStream`, and so on – weren't designed with subclassing in
+mind. And we've noticed that some users were trying to subclass them
+anyway, and ending up with fragile code that we're likely to
+accidentally break in the future, or else be stuck unable to make
+changes for fear of breaking subclasses.
+
+There are also some classes that were explicitly designed to be
+subclassed, like the ones in `trio.abc`. Subclassing these is still
```

Comment:
For what it's worth, the docs check is failing with `/home/runner/work/trio/trio/docs/source/history.rst:37:py:obj reference target not found: trio.abc`

Reply:
Ah, whoops, thanks for the heads up! I forgot we don't have a dedicated page for `trio.abc`.